test/integration/webhook/webhook.go

```diff
@@ -0,0 +1,171 @@
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package webhook
+
+import (
+	"context"
+	"path/filepath"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/gardener/landscaper/apis/config"
+	lsv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
+	hdv1alpha1 "github.com/gardener/landscaper/apis/deployer/helm/v1alpha1"
+	kutil "github.com/gardener/landscaper/pkg/utils/kubernetes"
+	"github.com/gardener/landscaper/test/framework"
+	"github.com/gardener/landscaper/test/utils"
+)
+
+// RegisterTests registers all tests of the package
+func RegisterTests(f *framework.Framework) {
+	SimpleWebhookTest(f)
+}
+
+func SimpleWebhookTest(f *framework.Framework) {
```

Comment:
```suggestion
	WebhookTest(f)
}

func WebhookTest(f *framework.Framework) {
```
I think we can omit the `simple`. I only used it in the tutorial test because the toturial is also called `SimpleNginx`